### PR TITLE
More themes and syntax styles

### PR DESCRIPTION
**Pull request by Geequlim**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -99,8 +100,10 @@ For installation instructions, read [INSTALL.md](https://github.com/Calinou/awes
 *Add-ons for text editors that add syntax highlighting to GDscript.*
 
 - [Atom](https://atom.io/packages/lang-gdscript)
+- [Chester](https://github.com/Geequlim/godot-themes/tree/master/syntax)
```
Comment (Calinou) [inline README.md:103]: Wrong section, please put Godot code editor themes in [this section](https://github.com/Calinou/awesome-godot#godot-text-editor-color-themes) (and, as of https://github.com/Calinou/awesome-godot/commit/85674242d6bd781861a407e34510f7a3d80866cf, it's already there).

Comment (Geequlim) [reply README.md:103]: They are the same project for now.

Comment (akien-mga) [reply README.md:103]: Oh right I merged too fast. @Geequlim This category is for external text editors, not for Godot's syntax highlighting themeing.

Comment (Geequlim) [reply README.md:103]: Oh I'm sorry for that.
Eidting it now.

**Pull request merged**